config/queue.php

```diff
@@ -37,7 +37,7 @@
         'database' => [
             'driver' => 'database',
             'table' => 'jobs',
-            'queue' => 'default',
+            'queue' => ['mail','default'],
```

Comment:
Would you be able to use the default queue for the mail please. (the production site can just one the default queue at the time being!)

Reply:
Amended in 82764e4.